src/rust/aggregator/src/pipelines/order_history_pipelines.rs

```diff
@@ -352,15 +352,15 @@ fn add_to_map<'a>(
         map.entry(LiquidityKey { group_id, bps_times_tem: bps_times_ten })
             .and_modify(|e| e.base += &order.size * &order.price)
             .or_insert(LiquidityValue {
-                base: &order.size * &order.price,
+                base: &order.size * price,
```

Comment:
I traced this logic back up the stack and see that it is coming from `get_cached_last_price`

Does this mean that during backfilling, the price used will be current last price, or the last price at the relevant time?

Reply:
It will always use the price at the relevant time. This is done so that you don't query the last price from the DB every time you need it. Since you might have 500 orders and one last price, there is no need to query the last price 500 times.